civihr_default_theme/scss/pages/_admin_absence_tab.scss

```diff
@@ -0,0 +1,47 @@
+.chr_leave-absences-admin {
+  background: transparent;
+  box-shadow: none;
+  padding: 0;
+}
+
+.chr_absence-tab-page {
```

Comment:
A lot of things are changed for the tabs which seems unnecessary to me as the tabs are same as manager and my leave. Only the active tab color and `button-container` margin seems different to me.